tests/transform/test_dynamodb.py

```diff
@@ -185,7 +192,9 @@ def test_decode_cdc_modify_basic():
 def test_decode_cdc_modify_nested():
     assert (
         DynamoCDCTranslatorCrateDB(table_name="foo").to_sql(MSG_MODIFY_NESTED) == 'UPDATE "foo" '
-        "SET data['tags'] = ['foo', 'bar'], data['empty_map'] = {}, data['empty_list'] = [] "
+        "SET data['tags'] = ['foo', 'bar'], data['empty_map'] = {}, data['empty_list'] = [],"
+        " data['locations'] = ['location_1'], "
+        "data['rules'] = [Decimal('1'), Decimal('2'), Decimal('3'), Decimal('4')] "
```

Comment:
CrateDB will probably not parse that SQL correctly, because it doesn't understand `Decimal('1')`, etc. Does it?

Reply:
Yep, missed that one